test/remote/gateways/remote_litle_test.rb

```diff
@@ -231,6 +231,42 @@ def test_authorization_capture_refund_void
     assert_equal 'Approved', void.message
   end
 
+  def test_authorization_capture_with_stored_credential
+    initial_stored_creds = {
+      stored_credential: {
+        initial_transaction: true,
+        reason_type: 'unscheduled',
```

Comment:
We should probably support initial transactions with a nil `reason_type`

Reply:
So the stored credential spec I'm implementing from seems to indicate that even for the initial transaction you need to set `processingType` to one of `initialRecurring`, `initialInstallment`, or `initialCOF`.

I'm not sure how we can satisfy that requirement if we don't have the user pass in the reason type even for that initial transaction. (See the spec PDF attached to ECS-138, page 2). Am I missing something there?

Reply:
Gotcha, that's fine. I don't think so, just trying to keep things permissive.

Reply:
Ok, cool. I don't think I'm actually enforcing this in the gateway adapter, so we should still be OK for being permissive (though like with not submitting the network_transaction_id, not submitting the reason_type will likely cause the gateway itself to raise some kind of error). Thanks again!